public/css/style-dark.scss

```diff
@@ -103,9 +103,14 @@ hr {
     text-align: center;
     font-size: 180px;
   }
+
+  #total-mishaps {
+    text-align: center;
+    font-size: 140px;
+  }
 }
 
-#chores li {
```

Comment:
I assume you tested this to make sure `#chores` captures `li` elements within it?

Reply:
I did, it works.